src/modules/custom.rs

```diff
@@ -32,6 +33,10 @@ pub fn module<'a>(name: &str, context: &'a Context) -> Option<Module<'a>> {
             is_match = exec_when(when, &config.shell.0);
         }
 
+        if let Some(os) = config.os {
+            is_match = os == env::consts::OS;
```

Comment:
I think this might cause some false positives since if `is_match` is already false e.g. the when command returned false this would override that and always display the module if the os matches.

Reply:
You're right, should be fixed now.